src/CountryHeader.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { CountryStoreContext } from './CountryStore';
+import './CountryHeader.sass';
+import { Tag } from 'antd';
+
+// const SECONDARY_COLOR = '#fee187';
+
+const CountryHeader = () => {
+    const { getDefaultCountryName } = useContext(CountryStoreContext);
+    const [country, setCountry] = useState(undefined);
+
+    useEffect(() => {
+        setCountry(getDefaultCountryName());
+    }, [getDefaultCountryName]);
+
+    return (
+        <div id="country-container">
+            <Tag style={{ fontSize: 'large', padding: 10 }} color="white">
```

Comment:
Move style to `.sass` file.
Maybe add a prefix: "You are now configuring:" or something like that